tests/LSP14Ownable2Step/LSP14Ownable2Step.behaviour.ts

```diff
@@ -127,6 +133,26 @@ export const shouldBehaveLikeLSP14 = (
         expect(accountBalanceAfter).to.be.lt(accountBalanceBefore);
       });
     });
+
+    describe("when the URD of the `newOwner` calls `acceptOwnership`", () => {
```

Comment:
I would suggest a different text description for this test.

```suggestion
    describe("when `transferOwnership(...)` is called in the same tx as `acceptOwnership(...)`", () => {
```

> **NB: ** also the text here states *when the URD of the `newOwner` calls `acceptOwnership`*. But this is not true, here the re-entrant callback comes from the UP via `universalReceiver(...)` function. 
> It's just a note, but we should double check that we describe the tests being run properly, to keep track correctly of what we actually test and don't test.